mrtt-ui/src/components/CheckboxGroup.js

```diff
@@ -0,0 +1,61 @@
+import { Checkbox, FormControlLabel, FormGroup } from '@mui/material'
+import PropTypes from 'prop-types'
+import { forwardRef, useEffect, useState } from 'react'
+
+const CheckboxGroup = forwardRef(({ options, value, onChange, onBlur }, ref) => {
+  const [checkboxItems, setCheckboxItems] = useState([])
+
+  const _loadCheckboxItems = useEffect(() => {
+    setCheckboxItems(value)
+  }, [value])
+
```

Comment:
I like to name my effects so people dont have to reverse engineer them to figure out what they do.

Reply:
curious why you are adding a hook for this? no need to use any state management with react hook form

Reply:
This is a generic component. It has no idea about the state of the parent, it just feeds back an array of selected values, and the original event and its composer can do whatever with it

I made another component to compose this one and tie in reach hook form.